scripts/archive/ocaml/stitch_introspection.ml

```diff
@@ -72,35 +67,32 @@ let remove_blocks_fee_transfers_input =
   remove_key_from_input ~type_name:"blocks_fee_transfers_insert_input" ["block_id"; "fee_transfer_id"]
   
 let remove_blocks_snark_jobs_input = 
-  remove_key_from_input ~type_name:"blocks_snark_jobs_insert_input" ["block_id"; "snark_job_id"]
+  remove_key_from_input ~type_name:"blocks_snark_jobs_insert_input" ["block_id"; "snark_job_id"] *)
 
-let change_constraint = 
+let change_constraint =
   let open Combinators in
   let predicate json =
     let open Option.Let_syntax in
     let open Pattern in
-        compute_validity
-      ( assoc_list json >>= ( key_and_value ~key:"name"
-              ~value:(String "constraint") ))
-    in
+    compute_validity
+      ( assoc_list json
+      >>= key_and_value ~key:"name" ~value:(String "constraint") )
+  in
   let action json =
     let open Change in
-    association_list json ~f:(value_with_name "name" ~f: (Fn.const (Or_error.return @@ `String "constraint_")))
+    association_list json
+      ~f:
+        (value_with_name "name"
+           ~f:(Fn.const (Or_error.return @@ `String "constraint_")))
   in
-    stitch ~predicate ~action
-
-let read_json json_string =
-  let json = Yojson.Basic.from_string json_string in
-  let cleaned_user_commands = remove_user_commands_input_commands json in
-  Core.print_string @@ Yojson.Basic.prettify
-  @@ Yojson.Basic.to_string cleaned_user_commands
+  stitch ~predicate ~action
 
 let read_from_standard_input () =
   (* There is only one async call in this entire program so it's okay to make this synchronous call *)
-  let json_string = Core.In_channel.input_all Core.In_channel.stdin in  
-  let json = Yojson.Basic.from_string json_string
-    |> change_constraint 
-    |> remove_public_keys_input
+  let json_string = Core.In_channel.input_all Core.In_channel.stdin in
+  let json =
+    Yojson.Basic.from_string json_string |> change_constraint
+    (* |> remove_public_keys_input
```

Comment:
any reason why commenting out instead of deleting?

Reply:
The entire file is already deleted